Live/Classes/MainScene.h

```diff
@@ -1,6 +1,9 @@
 #ifndef __MAIN_SCENE_H__
 #define __MAIN_SCENE_H__
 
+#include "cocos2d.h"
+#include "Player.h"
+#include "HUD/HUD.h"
```

Comment:
repeated includes again. Please do a review yourself first before making the PR.